react/features/invite/components/dial-in-info-page/DialInInfoApp.web.js

```diff
@@ -8,16 +8,19 @@ import { i18next } from '../../../base/i18n';
 import { DialInSummary } from '../dial-in-summary';
 
 import NoRoomError from './NoRoomError';
+import { Platform } from '../../../base/react';
 
 document.addEventListener('DOMContentLoaded', () => {
     const { room } = parseURLParams(window.location, true, 'search');
+    const OS = Platform.OS;
+    const isUsingMobileBrowser = OS === 'android' || OS === 'ios';
```

Comment:
An improvement could be to define the OS constants in the Platform object and compare with constants instead of strings.